src/Utility/External/LRUCache.js

```diff
@@ -143,8 +134,7 @@ class LRUCache {
             this.tail = node.prev;
         }
   
-        this._cache.delete(key);
-        this.size--;
+        this.cache.delete(key);
```

Comment:
```suggestion
        return this.cache.delete(key);
```
Store returns a boolean for this